Search.jsx

```diff
@@ -0,0 +1,26 @@
+function Search() {
+  const [userId, setUserId] = React.useState(1);
+  return (
+    <div className='search'>
+      <form>
+        <label>Users</label>
+        <select>
+          <option onClick={() => setUserId(1)} value='User 1'>User 1</option>
+          <option onClick={() => setUserId(2)} value='User 2'>User 2</option>
+          <option onClick={() => setUserId(3)} value='User 3'>User 3</option>
+          <option onClick={() => setUserId(4)} value='User 4'>User 4</option>
```

Comment:
One other thing to make your life easier - if you had an array of all the ids, you could map over that and render out an option for each one. 

```jsx
const userIds = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12]
<select>
  { userIds.map((userId) => <option value={userId}>User {userId}</option>)}
</select>
```

You could even imagine pulling that array in from somewhere else as well, like the API itself or something. Or, you could use a little trick to build up the array programmatically ( I looked this up on Stack Overflow)

```js
const userIds = Array(12).fill().map((_, i) => i _+ 1); // get an array with a length of 12, fill it with undefined, then map over an use the index to get the value
  
```